values.yaml

```diff
@@ -147,11 +147,13 @@ deploymentAnnotations: {}
 podLabels: {}
 
 # -- (object) The securityContext to use for the krakend pod
-podSecurityContext: {}
+podSecurityContext:
+  enabled: false
   # fsGroup: 2000
 
 # -- (object) The securityContext to use for the krakend container
 securityContext:
+  enabled: true
```

Comment:
Let's not add these toggles here, given the usage, it breaks compatibility and users would need to change how they call these. Feel free to create a `podSecurityContextEnabled` and a `securityContextEnabled` key that's separate from the main object.